apps/etterlatte-behandling/src/main/kotlin/inntektsjustering/AarligInntektsjusteringJobbService.kt

```diff
@@ -76,13 +78,46 @@ class AarligInntektsjusteringJobbService(
 ) {
     private val logger = LoggerFactory.getLogger(this::class.java)
 
-    fun startAarligInntektsjustering(request: AarligInntektsjusteringRequest) {
+    // for å starte inntektsjusteringJobb for alle saker som er løpende i neste inntektsår
+    fun startAarligInntektsjusteringJobb(request: AarligInntektsjusteringRequest) {
+        logger.info("Starter årlig inntektsjusteringjobb $kjoering")
         request.saker.forEach { sakId ->
             startEnkeltSak(request.kjoering, request.loependeFom, sakId)
         }
     }
 
-    fun opprettManuellInntektsjustering(
+    // for å behandle inntektsjustering via innsendt skjema
+    fun startInntektsjusteringJobb(request: InntektsjusteringRequest) {
```

Comment:
Tror det er lurt med en egen service for inntektsjusteringer som er innkommende fra selvbetjening. Vi vil jo på sikt tilby skjema for inneværende år også og da er det jo ikke en del en "årlig jobb".